packages/code-du-travail-api/tests/cdtn_document.data.json

```diff
@@ -104,521 +104,7 @@
-      0.0497615672647953,
-      0.02424652688205242,
-      -0.0008102610590867698,
-      -0.023710576817393303,
-      -0.018190186470746994,
-      0.046574272215366364,
-      0.018023313954472542,
-      0.029168371111154556,
-      0.042810168117284775,
-      -0.0025900728069245815,
-      -0.029405508190393448,
-      -0.053939174860715866,
-      -0.06019484996795654,
-      0.047622911632061005,
-      0.0305231474339962,
-      -0.05351817235350609,
-      -0.04538353160023689,
-      0.016736306250095367,
-      -0.054554931819438934,
-      0.042899712920188904,
-      -0.03252421319484711,
-      -0.024979332461953163,
-      0.0691576823592186,
-      -0.03345945104956627,
-      -0.04362364858388901,
-      -0.033257629722356796,
-      0.018124060705304146,
-      0.059635475277900696,
-      0.03952322155237198,
-      0.054619088768959045,
-      -0.04323865845799446,
-      0.07555671781301498,
-      -0.04616973549127579,
-      0.04946132376790047,
-      -0.07444567233324051,
-      0.0036138235591351986,
-      -0.02731567807495594,
-      -0.0471782460808754,
-      0.04201934486627579,
-      0.053163547068834305,
-      -0.052573949098587036,
-      -0.023611033335328102,
-      -0.03535439074039459,
-      0.011948046274483204,
-      0.07342529296875,
-      -0.04518789425492287,
-      0.025681819766759872,
-      0.034935228526592255,
-      -0.05266878381371498,
-      0.032839950174093246,
-      0.040536414831876755,
-      0.07884977757930756,
-      0.04387161135673523,
-      0.030873913317918777,
-      -0.006429867818951607,
-      0.08034439384937286,
-      0.0396617092192173,
-      0.007274125702679157,
-      -0.03976615145802498,
-      0.05385816842317581,
-      -0.04962356388568878,
-      -0.032899968326091766,
-      -0.037649549543857574,
-      0.02850538119673729,
-      -0.049477510154247284,
-      -0.023860469460487366,
-      0.05042380839586258,
-      -0.03391316160559654,
-      -0.04607649892568588,
-      0.057806726545095444,
-      0.017131265252828598,
-      0.038717858493328094,
-      0.08263079822063446,
-      -0.003942811395972967,
-      -0.03660926595330238,
-      -0.021345259621739388,
-      0.04090287536382675,
-      0.02316957153379917,
-      0.04130781441926956,
-      -0.0265121441334486,
-      -0.07781603187322617,
-      0.03328760713338852,
-      0.023601777851581573,
-      0.006731935776770115,
-      -0.0016169104492291808,
-      0.016610639169812202,
-      -0.0562560148537159,
-      -0.009146842174232006,
-      -0.08039947599172592,
-      -0.06989061832427979,
-      0.04550381377339363,
-      0.042644333094358444,
-      -0.02738923951983452,
-      -0.050818201154470444,
-      -0.04442508518695831,
-      0.0005298219621181488,
-      -0.0331767275929451,
-      0.07868209481239319,
-      -0.01221694890409708,
-      0.047598399221897125,
-      0.027510926127433777,
-      0.049646858125925064
-    ]
+    "excludeFromSearch": false
```

Comment:
you need to add title_vector back please to avoid breaking to much api tests

Reply:
Yep my bad, they broke and I was out of idea on why did my test passed locally but not on the CI so I tried to regenerate the data, and it worked but I clearly missed that :o

Reply:
np, if you need help to update exported data, you can follow readme, it should be up to date !